compatibility-tests/compile-fail/tests/ui/attribute-duplication.stderr

```diff
@@ -0,0 +1,71 @@
+error: Only one field can be designated as 'visibility'; found a duplicate here:
```

Comment:
This isn't a field, is it? Shouldn't this say something like "multiple visibility attributes are not supported on an error enum"?

Reply:
You're right, the context isn't as good as it should be, even without more detailed spans.  I'll work on improving these.